azure/services/privateendpoints/spec_test.go

```diff
@@ -356,14 +229,14 @@ func TestParameters(t *testing.T) {
 			g := NewWithT(t)
 			t.Parallel()
 
-			result, err := tc.spec.Parameters(context.TODO(), tc.existing)
+			result, err := tc.spec.Parameters(context.TODO(), tc.existing.DeepCopy())
```

Comment:
FYI this particular `DeepCopy` isn't strictly necessary if `tc.existing` isn't being read after calling `Parameters`.